cmd/root.go

```diff
@@ -18,22 +18,65 @@ import (
 	"fmt"
 	"os"
 
+	"github.com/gophergala2016/reviewer/reviewer"
 	"github.com/spf13/cobra"
 	"github.com/spf13/viper"
 )
 
 var cfgFile string
 
-// This represents the base command when called without any subcommands
+type config struct {
+	Authorization struct {
+		Token string
+	}
+	Repositories map[string]struct {
+		Username string
+		Status   bool
+		Required int
+	}
+}
```

Comment:
I think that we should review @alvarocabanas implementation for the configuration unmarshaling. Not sure if he did that way.